tools/configuration-validator/main.go

```diff
@@ -0,0 +1,61 @@
+package main
+
+import (
+	"io/ioutil"
+	"log"
+	"os"
+
+	yamlproto "github.com/envoyproxy/xds-relay/internal/pkg/util"
+	aggregationv1 "github.com/envoyproxy/xds-relay/pkg/api/aggregation/v1"
+
+	"github.com/spf13/cobra"
+)
+
+type KeyerConfiguration = aggregationv1.KeyerConfiguration
+
+var (
+	cfgFile string
+
+	validatorCmd = &cobra.Command{
+		Use:   "configuration-validator",
```

Comment:
Can you make configuration-validator the root command but add a `aggregation` subcommand? So that we can extend it to support other configurations.